tests/commands/test_rm.py

```diff
@@ -39,6 +41,7 @@ def test_rm_files_run(tmp_library: TemporaryLibrary) -> None:
     run(doc, filepath=filename, git=True)
     assert not os.path.exists(filename)
 
+    assert isinstance(db, papis.database.cache.Database)
```

Comment:
Is this needed because it sets `db.documents = None`? This should be already done in `Database.clear`, so you can remove that line.

Reply:
That works.